ml/notebooks/netdata_anomaly_detection_deepdive.ipynb

```diff
@@ -0,0 +1,1705 @@
+{
```

Comment:
<p>I would start this cell with a basic of explanation of what we are doing next.</p><p>Something like: "Now that we have set up our working environment, we can begin to pull in data. For now, we will focus on basic parameters to choose which data to pull and the most important ML parameters:</p><p>(List of parameters)</p><p> </p><p>Note: You can find detailed descriptions of all the parameters above and more in the <a href="https://github.com/netdata/netdata/blob/master/ml/README.md#configuration" target="_blank">ML readme | Configuration</a></p><p> </p><p> </p><p>Feedback for the list: </p><ul><li>Capitalize "Netdata Agent"</li><li>Capitalize acronyms, such as API </li><li>Avoid filler words in technical descriptions ("really just")</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netdata/netdata/pull/12313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>